src/Symfony/Component/DependencyInjection/Loader/Configurator/ContainerConfigurator.php

```diff
@@ -118,31 +118,48 @@ function iterator(array $values): IteratorArgument
  *
  * @deprecated since Symfony 4.4, to be removed in 5.0, use "tagged_iterator" instead.
  */
-function tagged(string $tag, string $indexAttribute = null, string $defaultIndexMethod = null): TaggedIteratorArgument
+function tagged(string $tag, string $indexAttribute = null, string $defaultIndexMethod = null, bool $needsIndexes = false, string $defaultPriorityMethod = null): TaggedIteratorArgument
 {
     @trigger_error(__NAMESPACE__.'\tagged() is deprecated since Symfony 4.4 and will be removed in 5.0, use '.__NAMESPACE__.'\tagged_iterator() instead.', E_USER_DEPRECATED);
 
-    return new TaggedIteratorArgument($tag, $indexAttribute, $defaultIndexMethod);
+    return new TaggedIteratorArgument($tag, $indexAttribute, $defaultIndexMethod, $needsIndexes, $defaultPriorityMethod);
 }
 
 /**
  * Creates a lazy iterator by tag name.
+ *
+ * @param string $tag
+ * @param string|null $indexAttribute
+ * @param string|null $defaultIndexMethod
+ * @param bool $needsIndexes
+ * @param string|null $defaultPriorityMethod
+ * @return TaggedIteratorArgument
```

Comment:
should be reverted unless you need to add some useful description (same bellow)

Reply:
Same I'll fix it